app/components/users-cards.js

```diff
@@ -1,4 +1,36 @@
 import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed, get, set } from '@ember/object';
 
 export default Component.extend({
+  store: service(),
+
+  allUsers: computed(function () {
+    return get(this,'store').findAll('user', { reload: true });
+  }),
+
+  userList: computed('getTeam', function () {
+    if (this.getTeam) {
+      return this.allUsers.filterBy('team', this.getTeam);
+    } else {
+      return get(this,'store').findAll('user', { reload: true });
```

Comment:
`return this.getTeam ? this.allUsers.filterBy('team', this.getTeam) : this.allUsers;`